File: Python/boj/Silver/5648.py
import sys

if __name__ == "__main__":
    try:
        sys.stdin = open("input.txt", "r")
    except FileNotFoundError:
        pass

    input = list(map(int, sys.stdin.read().split()))
    result = []
    n = input.pop(0)

    for number in input:
        numberList = list(str(number))
        numberList.reverse()
        result.append(int("".join(numberList)))

    print("\n".join(map(str, sorted(result))))

File: JavaScript/boj/Silver/5648.js
const fs = require('fs');
const filePath = process.platform === 'linux' ? '/dev/stdin' : './input.txt';
const input = fs
  .readFileSync(filePath)
  .toString()
  .trim()
  .split('\n')
  .filter((value) => value !== '');

const numbers = [];
const result = [];

for (let i = 0; i < input.length; i++) {
  const line = input[i].split(' ');

  for (let i = 0; i < line.length; i++) numbers.push(line[i]);
}

for (let i = 1; i <= +numbers[0]; i++) {
  const number = Number(numbers[i].split('').reverse().join(''));

  result.push(number);
}

console.log(result.sort((a, b) => a - b).join('\n'));
